apps/src/templates/CodeWorkspaceContainer.jsx

```diff
@@ -97,7 +97,7 @@ var CodeWorkspaceContainer = React.createClass({
   }
 });
 module.exports = connect(state => ({
-  hidden: state.pageConstants.hideSource,
+  hidden: state.pageConstants.hideSource && !state.pageConstants.visualizationInWorkspace,
```

Comment:
The `CodeWorkspaceContainer` won't be hidden in `hideSource` mode for apps where the visualization is in the workspace.